mne/preprocessing/ica.py

```diff
@@ -2255,170 +2255,6 @@ def _detect_artifacts(ica, raw, start_find, stop_find, ecg_ch, ecg_score_func,
     logger.info('Ready.')
 
 
-@verbose
-def run_ica(raw, n_components, max_pca_components=100,
```

Comment:
We should have a one-release deprecation cycle for release. You can probably do this by, instead of deleting this code, decorate it with the `deprecated` decorator with a suitable message like:
```
@deprecated('run_ica is deprecated and will be removed in 0.21, use ica.detect_artifacts() instead')
@verbose
def run_ica(...)
```